src/rust/engine/src/cffi_build.rs

```diff
@@ -0,0 +1,7 @@
+extern crate gcc;
+
```

Comment:
This is fairly clear, but a snippet/blurb (basically, some of the review description) would be good inline.